http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -200,13 +211,27 @@ public JettyHttpClient(
             Tracer tracer,
             Optional<String> environment,
             Optional<SslContextFactory.Client> maybeSslContextFactory)
+    {
+        this(name, config, requestFilters, openTelemetry, tracer, environment, maybeSslContextFactory, ImmutableList.of());
+    }
+
+    public JettyHttpClient(
```

Comment:
As a follow up, we should probably deprecate most/all of these constructors and add a builder